odc/stats/io.py

```diff
@@ -229,8 +229,7 @@ def _apply_color_ramp(
 
         ows_style = StandaloneStyle(ows_style_dict)
         # assign the time to xr.Dataset cause ows needs it
```

Comment:
Seems like this comment is now outdated?

Reply:
the code still does what's in the comment, only differently, to be compatible with newer xarray